packages/yoroi-ergo-connector/src/inject.js

```diff
@@ -139,6 +114,8 @@ class ErgoAPI {
 
 const ergo = Object.freeze(new ErgoAPI());
 `
+const API_INTERNAL_ERROR = -2;
```

Comment:
Three things:
- Maybe these error codes should be documented somewhere? 
- Maybe these should should be in an enum instead of two separate variables
- Why does it start at -2 instead of -1? Is -1 used for something else? Maybe document this

Reply:
https://github.com/ergoplatform/eips/pull/23 they're documented here in the EIP-0012 spec. They do start at -1, but we just don't reject things for `InvalidRequest: -1` in the connector since input data verification happens elsewhere.